### ISSUE TITLE
Default Map view should display all filtered Projects.

### ISSUE THREAD

**Issue by LAlexMHllc** [MicroHealthLLC/mGis#1688]
User clicked the Map tab and 11 Projects displayed instead of all filtered Projects.
Screenshot Example

![map_initial_view_projects_11](https://user-images.githubusercontent.com/11938477/107244727-5691c180-69fc-11eb-91ef-7e4a1881c64e.png)

User clicked zoom out and 13 Projects displayed.
Screenshot Example

![map_zoom_out_projects_13](https://user-images.githubusercontent.com/11938477/107245073-bdaf7600-69fc-11eb-8403-49c44593126a.png)

**Comment by christopherandrewcalderon** [comment]
@LAlexMHllc I looked into this issue and I found the reason why the map is not showing all projects on load is due to the width of the map being too short for the zoom level the map is trying to achieve.

The ratio of map to sidebar was recently reduced from 8-4 to 7-5. I changed the ratio back to 8-4 as well as reducing the padding around the markers and the map was able to display all markers from Alaska to New York such as in the images above:

![image](https://user-images.githubusercontent.com/77029475/107408284-9e2d5180-6abf-11eb-92e4-bb91ba1c7b3d.png)

My view-port width is 1375px. With a higher view-port width the issue is less likely to occur (based on my testing locally). I would suggest converting back to a 8-4 ratio. I will discuss width @juan-rivera-2020 who changed the ratio to 7-5. I will also implement the reduced padding.
